### PR TITLE
Add Rust workshop to calendar

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- created md file
- added image

### CODE REVIEW

**Review by marcoow — COMMENTED** [review]
I realize we currently only render conferences and meetups on the calendar page – I'll update the page so that this actually shows up.

File: _calendar/2022-05-10-rust-workshop.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+url: https://ti.to/simplabs/rust-workshop-for-beginners-may-2022/with/rust-workshop-for-beginners
+kind: workshop
+---
+
+Rust has become one of the most popular new programming languages in recent
+years. The combination of performance on the level of C++ with stability and
+security on the level of managed languages like C# or Java is attractive to many
+development teams.
+
+The workshop will take place from the 10th to the 13th of May, from 1pm to 6pm.
```
Comment (marcoow) [inline _calendar/2022-05-10-rust-workshop.md]:
```suggestion
Rust has become one of the most popular new programming languages in recent
years. The combination of performance on the level of C++ with stability and
security on the level of managed languages like C# or Java is attractive to many
development teams. This workshop offers participants an introduction to the world of Rust. It takes place over 5 afternoons, from the 10th to 13th of May, 13:00 to 18:00 CET each day.
```

**Review by marcoow — COMMENTED** [review]
We need to use a much smaller image – this is how it will render:

Bildschirmfoto 2022-02-22 um 20 14 25

**Comment by emmasofiah2o** [comment]
> We need to use a much smaller image – this is how it will render:
> 
> Bildschirmfoto 2022-02-22 um 20 14 25

So we need to make a visual with no text I assume?

**Comment by emmasofiah2o** [comment]
Would this work?
Screenshot 2022-02-23 at 11 21 49

**Comment by emmasofiah2o** [comment]
I have two options in the folder. I also made a bigger one cause I think it looks better. Let me know which one I should go for or if you had something different in mind @marcoow 
Screenshot 2022-02-23 at 11 26 00

**Comment by marcoow** [comment]
We shouldn't alter colors of third party logos. White with our blue as a background or so should be fine I guess.

**Comment by emmasofiah2o** [comment]
Screenshot 2022-02-23 at 11 44 49
How about this?

**Comment by marcoow** [comment]
sure, that looks good 👍

**Review by marcoow — APPROVED** [review]

**Pull request merged**